### PR TITLE
DEVX-935: explain in README where to find the MDS logs

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: security/rbac/README.md [hunk 1]
```diff
@@ -48,13 +48,30 @@ $ ./enable-rbac-rest-proxy.sh
...
 $ ls ../delta_configs/
 
 # The modified configuration = original + delta
 $ ls /tmp/rbac_configs/
 ```
 
+4. After you run the demo, view the log files for each of the services. Since this demo uses Confluent CLI, all logs are saved in a temporary directory specified by `confluent local current`.
+
+```bash
+$ ls `confluent local current | tail -1`
```
Comment (rspurgeon) [inline security/rbac/README.md:61]: From local testing, the pipe to `tail -1` here doesn't seem necessary.  I'm guessing the output of `local current` goes to stdout.  The "local commands..." printed message goes to stderr.  So the `ls` command works here w/ out the pipe.   

side note, I was supresseing the warning with the following:
```bash
➜ confluent local current 2> /dev/null
/var/folders/vc/_h2l6_yd0hbdk__7vz2rs4180000gp/T/confluent.mMxYbM5S
```

Comment (ybyzek) [reply security/rbac/README.md:61]: @rspurgeon the previous version of `confluent` CLI required the `tail`.  Here is the output of it not working and working with CP 5.2.2:

```bash
code: ls `confluent current`
ls: cannot access 'This': No such file or directory
ls: cannot access 'CLI': No such file or directory
ls: cannot access 'is': No such file or directory
ls: cannot access 'intended': No such file or directory
ls: cannot access 'for': No such file or directory
ls: cannot access 'development': No such file or directory
ls: cannot access 'only,': No such file or directory
ls: cannot access 'not': No such file or directory
ls: cannot access 'for': No such file or directory
ls: cannot access 'production': No such file or directory
ls: cannot access 'https://docs.confluent.io/current/cli/index.html': No such file or directory
/var/folders/xn/c8wt_2_x57bczxjkpgcd13pr0000gq/T/confluent.34qN5CFx:
total 0
drwx------   8 yeva staff  256 Jul 16 22:28 .
drwx------ 211 yeva staff 6752 Jul 17 12:09 ..
drwxr-xr-x   7 yeva staff  224 Jul 16 22:27 connect
drwxr-xr-x   7 yeva staff  224 Jul 16 22:26 kafka
drwxr-xr-x   7 yeva staff  224 Jul 16 22:28 kafka-rest
drwxr-xr-x   7 yeva staff  224 Jul 16 22:28 ksql-server
drwxr-xr-x   7 yeva staff  224 Jul 16 22:27 schema-registry
drwxr-xr-x   7 yeva staff  224 Jul 16 22:26 zookeeper


code: ls `confluent current | tail -1`
total 0
drwx------   8 yeva staff  256 Jul 16 22:28 .
drwx------ 211 yeva staff 6752 Jul 17 12:09 ..
drwxr-xr-x   7 yeva staff  224 Jul 16 22:27 connect
drwxr-xr-x   7 yeva staff  224 Jul 16 22:26 kafka
drwxr-xr-x   7 yeva staff  224 Jul 16 22:28 kafka-rest
drwxr-xr-x   7 yeva staff  224 Jul 16 22:28 ksql-server
drwxr-xr-x   7 yeva staff  224 Jul 16 22:27 schema-registry
drwxr-xr-x   7 yeva staff  224 Jul 16 22:26 zookeeper
```

The new version of the Confluent CLI does not need the `tail`, thank you for observing that.  We need to update all 5.3.x examples to match this new behavior (not just this demo), and I have filed DEVX-961 to track this separately.

Comment (rspurgeon) [reply security/rbac/README.md]: Yeah, was going to speculate given feedback that they may have moved the warning to stderr recently

**Review by rspurgeon — APPROVED** [review]
LGTM

**Pull request merged**